database/insertData/03_template_E_drugRegistration.js

```diff
@@ -541,7 +912,6 @@ exports.queries = [
                 }
               }
               {
-                id: 10
```

Comment:
Can you add `${devActions}` just below `${coreActions}` and import it at the top? (see other templates)

Reply:
Done: ecfbfcb